tools/download-ravendb-server.ps1

```diff
@@ -51,3 +51,6 @@ Copy-Item -Path $serverPath -Destination $deployPath -Recurse
 Write-Output "Deleting temporary files"
 Remove-Item $zipPath -Force
 Remove-Item $unzipPath -Force -Recurse
+
```

Comment:
Given that the above test outputs stuff when it fails and locally you can execute the script and do a diff of the folder I would suggest removing this